plugins/radar-android-google-places/src/main/java/org.radarbase.passive.google.places/GooglePlacesService.kt

```diff
@@ -52,6 +58,7 @@ class GooglePlacesService: SourceService<GooglePlacesState>() {
             start()
         }
         preferences = getSharedPreferences(GooglePlacesService::class.java.name, Context.MODE_PRIVATE)
+        _broadcaster = LocalBroadcastManager.getInstance(this)
```

Comment:
Initialize the BroadcastManager after the service instance is created to correctly get the context.